docs/rules/no-dynamic-imports-in-ssrable-components.md

```diff
@@ -0,0 +1,19 @@
+# Disallow dynamic imports in SSR-able components (`lwc/no-dynamic-imports-in-ssrable-components`)
+
+Dynamic imports are not processed on the server and do not affect server-side rendered output.
+
+## Rule details
+
+Example of **incorrect** code:
+
+```js
+import { LightningElement } from 'lwc';
+
+export default class Foo extends LightningElement {
+    connectedCallback() {
+        import('@salesforce/user/isGuest').then((module) => {
```

Comment:
It is odd that the recommended solution solution for `no-static-imports-of-user-specific-scoped-modules-in-ssrable-components` conflicts with the `no-dynamic-imports-in-ssrable-components`.

Also looking at the implementation at the rule implementation, there is no check if a module is server-side renderable. 

My 2 cents. This rule will certainly introduce more confusion than anything else. Reporting the usage of dynamic import is a half measure and doesn't solve the underlying issue.

Reply:
Remove the rule regarding the dynamic imports.

Reply:
>My 2 cents. This rule will certainly introduce more confusion than anything else. Reporting the usage of dynamic import is a half measure and doesn't solve the underlying issue.

@pmdartus 

Are the current recommendations for using DataProviders sufficient to solve the underlying issue?

Does removing the dynamic imports eslint rule resolve your concern?

Reply:
> Does removing the dynamic imports eslint rule resolve your concern?
Yes it does. Thank you.